etc/kayobe/ansible/purge-command-not-found.yml

```diff
@@ -11,7 +11,7 @@
 # by implementing this playbook separately for each group.
 - name: Purge command-not-found package
   hosts: seed-hypervisor:seed:overcloud:infra-vms
-  gather_facts: false
+  gather_facts: true
```

Comment:
Gathering facts as the bootstrap user can be a bad idea if using fact caching because we later reference facts about the user.

Can we change to use `os_distribution` instead?